### DISCUSSION TITLE
Remove Wild Crops from Mushroom Islands

### DISCUSSION THREAD

**Discussion by voidsong-dragonfly** [vectorwing/FarmersDelight#562] [Ideas]
Currently, wild crops such as wild onions generate in Mushroom Island biomes - something that doesn't make much sense as all life on the island is supposed to be mycelial. Removing these generations and instead replacing them with something mushroom-oriented if possible would be a better option to keep the fungal feel of the island.

**Comment by vectorwing** [comment]
Hello! This is already the case in version 1.2 of Farmer's Delight.

Previously, Wild Onion and Wild Carrot would mistakenly spawn in Mushroom Fields due to it also being a temperate climate. In 1.2, these biomes are deny-listed; instead, FD spawns Mushroom Colonies there, in varying stages of maturity.

**Reply by voidsong-dragonfly** [reply]
Thanks so much, looks like I must've genned these chunks with 1.1.2 - sorry for bothering you!

**Reply by vectorwing** [reply]
No problem! Hope you enjoy the update! Let me know if anything isn't working as it should. ^^
